lib/matplotlib/projections/polar.py

```diff
@@ -840,9 +864,25 @@ def _set_lim_and_transforms(self):
 
         # The complete data transformation stack -- from data all the
         # way to display coordinates
+        #
+        # 1. Remove any radial axis scaling (e.g. log scaling)
+        # 2. Shift data in the theta direction
+        # 3. Project the data from polar to cartesian values
+        #    (with the origin in the same place)
+        # 4. Scale and translate the cartesian values to Axes coordinates
+        #    (here the origin is moved to the lower left of the Axes)
+        # 5. Move and scale to fill the Axes
+        # 6. Convert from Axes coordinates to Figure coordinates
```

Comment:
I do not get this at all because of lacking background, but is there a particular reason that the last three terms are within brackets and is that reason already mentioned?

Reply:
At a guess because the bit in brackets is the full affine part of the transform.